src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -559,29 +561,39 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         if (projectId) {
             query = query.where({ project: projectId });
         }
-
-        if (queryString?.trim()) {
+        const hasQueryString = Boolean(queryString?.trim());
+        const hasHalfTag = normalizedHalfTag && normalizedHalfTag.length > 0;
+        if (hasQueryString || hasHalfTag) {
+            const tagQuery = this.db.from('feature_tag').select('feature_name');
             // todo: we can run a cheaper query when no colon is detected
-            const tagQuery = this.db
-                .from('feature_tag')
-                .select('feature_name')
-                .whereRaw("(?? || ':' || ??) LIKE ?", [
+            if (hasQueryString) {
+                tagQuery.whereRaw("(?? || ':' || ??) ILIKE ?", [
                     'tag_type',
                     'tag_value',
                     `%${queryString}%`,
                 ]);
+            }
+            if (hasHalfTag) {
```

Comment:
Conceptually, what is a "half tag"?

Reply:
@FredrikOseberg  our tags consist of **type** and **value**. So a full tag would be the complete tag consisting of type and value. So `simple:mytag`, but previously we had possibility to search just by value or by type, without providing full tag definition. Just by writing simple, we got all `simple` tags.